content/docs/user-guide/basic-concepts/dvc-project.md

```diff
@@ -12,7 +12,36 @@ match:
     repositories
   ]
 tooltip: >-
-  Initialized by running `dvc init` in the **workspace** (typically a Git
-  repository). It will contain the `.dvc/` directory, as well as `dvc.yaml` and
-  `.dvc` files created with commands such as `dvc add` or `dvc run`.
+  A DVC project is a directory initialized by `dvc init` that contains a `.dvc`
+  directory. It contains all <abbr>metafiles</abbr> related to project and also
+  includes the underlying <abbr>cache</abbr>.
+  [📖](/doc/user-guide/basic-concepts/dvc-project)
 ---
+
+# DVC Project
+
+A DVC project (or DVC repository) is a directory initialized by `dvc init`. It
+contains [`.dvc` directory](/doc/user-guide/project-structure/internal-files),
+`dvc.yaml` files that describe <abbr>pipelines</abbr> and `*.dvc` files for each
+tracked data file. Files in <abbr>remotes</abbr> and <abbr>cache</abbr>, and
+files like <abbr>metrics</abbr> and <abbr>plots</abbr> produced by DVC are also
```

Comment:
- I don't think we consider files in remote storage part of the project usually (they relate but I wouldn't mention it). Remotes are a separate concept: a backup and a way to share cached data 
- Agree to emphasize metrics/plots 👍  but lets then also mention parameters I think. 
- However we're missing mentioning general git-tracked files like code?